.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+## Motivation
```

Comment:
For me, `Motivation` and `Why` are the same. I often delete one or the other.

Reply:
I agree.

Reply:
I agree, but is the default template used. https://github.com/aerogear/standards
I think we should change there and replicate the changes for all projects if it is the case.